.github/workflows/cmake.yml

```diff
@@ -52,19 +52,27 @@ jobs:
 
       - name: Run .NET tests
         working-directory: ${{github.workspace}}/dotnet
-        run: dotnet test /p:EnableWindowsTargeting=true
+        run: dotnet test --collect:"XPlat Code Coverage" --results-directory ${{github.workspace}}/coverage_imgdoc2net /p:EnableWindowsTargeting=true
```

Comment:
Hmmm... not sure "how this actually works", or - do we get coverage for the managed code or the unmanaged code or both? If unmanaged code is included - shouldn't we then use a debug-build? But well - let's give it a shot and find out.

Reply:
Good point. Will do as part of #24 .